src/dags/bbga.py

```diff
@@ -35,11 +36,14 @@
 data_endpoints: Dict[FileStem, UrlPath] = VARS["data_endpoints"]
 table_mappings: Dict[FileStem, str] = VARS["table_mappings"]
 
+# We have one extra table that does not map to a downloadable dataset
+# Hence we add it explicitly here.
+TABLES: Tuple[str, ...] = tuple(chain(table_mappings.values(), ("bbga_cijfers",)))
```

Comment:
Surely this is not performance-critical. Can we use `+` concatenation for readability?

```suggestion
TABLES: Tuple[str, ...] = tuple(table_mappings.values()) + ("bbga_cijfers",)
```

Reply:
Wasn't after performance. This is just how list/tuple concatenation based code springs into existence in my brain. 😉

Will change and amend existing commit once I'm back at my laptop